codespell_lib/data/dictionary.txt

```diff
@@ -44943,7 +45070,13 @@ reverseed->reversed
 reversees->reverses
 reverve->reserve
 reverved->reserved
+revew->review
```

Comment:
Also `renew`, `resew`, `revel`, `revex`?

Reply:
I'll add `renew` and `revel`.